src/events/index.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+
+const ElementHandlers: WeakMap<Element, Map<string, Function>> = new WeakMap();
+
+const returnsFalse = () => false;
+const returnsTrue = () => true;
+const persist = function() {
+  this.isPersistent = returnsTrue;
+};
+const isPropagationStopped = function() {
+  return this.cancelBubble;
+};
+const isDefaultPrevented = function() {
+  return this.defaultPrevented;
+};
+
+const handlerKey = (name, capturing) => `${name}-${String(capturing)}`;
+
+const shouldUseCapture = name =>
+  name === 'focus' ||
+  name === 'blur' ||
+  name === 'scroll' ||
+  name === 'cancel' ||
+  name === 'close';
```

Comment:
i'll admit i cargo-culted this a bit, since i really don't know much about `cancel` or `close` events.

Reply:
Does dom-lite do synthetic event bubbling? Reason I ask: I think all delegated events should use capture anyway. It prevents the need to attach local listeners for media events and other things.

Reply:
(I've been wanting to check this out in React, too)

Reply:
the goal is compatible with react-dom behavior modulo a different set of browsers to support, and stuff that's too hard. In this case I'd support bubbling if it's a s simple as using capture.  We aren't doing any top level delegation atm, so maybe this list need to be bigger?

Reply:
generally I think react-dom but only for (modern) mobile browsers

Reply:
Media events, error, and load do not bubble. Basically anything that we attach locally in ReactDomFiberComponent.

Sending this from my phone, otherwise I'd send a link! I tested that all of these events register via capture on the document in every browser

Reply:
I wonder what the cost of mimicking the synthetic bubbling/capturing properly is... it seems like tree traversal would get muddy with local handlers (and no delegation) but i guess it's easy enough to do `this === event.target`